docs_src/microservices/device/supported/device-usb-camera/walkthrough/deployment.md

```diff
@@ -109,6 +109,74 @@ Follow this guide to deploy and run the service.
     !!! Note 
         The `jq -r` option is used to reduce the size of the displayed response. The entire device with all information can be seen by removing `-r '"deviceName: " + '.devices[].name'', and replacing it with '.'`
 
+1. Add credentials to for the rtsp stream.
+
+    !!! note
+        If you want to disable rtsp authentication entirely, you must [build a custom image](../walkthrough/custom-build.md).
+
+    === "Scripts"
```

Comment:
i dont think this points to the scripts. also we may not want to put too much work into the secret documentation if I get https://github.com/edgexfoundry/go-mod-bootstrap/pull/542 merged in